cypress/integration/inventory/export.spec.js

```diff
@@ -37,4 +38,14 @@ describe('inventory / actions: export UUIDs', () => {
         });
     });
   });
+
+  it('C196755 verifies search result counts and selected counts', () => {
```

Comment:
Scroll down so that records selected in the first step are not visible anymore  - may be this portion of manual test is required, please check

Reply:
Thanks
But I found it excess for this test because I check alredy ckeck whole string with search and selecter results